supervisor/docker/addon.py

```diff
@@ -709,6 +709,33 @@ async def import_image(self, tar_file: Path) -> None:
             with suppress(DockerError):
                 await self.cleanup()
 
+    @Job(name="docker_addon_cleanup", limit=JobExecutionLimit.GROUP_WAIT)
+    async def cleanup(
+        self,
+        old_image: str | None = None,
+        image: str | None = None,
+        version: AwesomeVersion | None = None,
+    ) -> None:
+        """Check if old version exists and cleanup other versions of image not in use."""
+        if not image:
+            image = self.image
+        if not version:
+            version = self.version
```

Comment:
This is logically equivalent to what we have in `supervisor/docker/interface.py:cleanup()` right? If so, can we use the same syntax? Either way works for me, I just prefer to have the same in both functions.

Reply:
Yes. The only reason I did it that way here is because I used the value of `image` twice in here. In the other one I only used it once as the value of the parameter so I just did `image or self.image` because it was more concise.

The version part can be reverted. I was using `version` in the loop conditional as well at one point and then changed my mind. There's no good reason it doesn't use `version or self.version` here anymore.

I guess I can change the other to use this syntax for image if we want. I wasn't really too bothered by it personally since this one doesn't call the `super` anymore anyway.